.github/workflows/load-db-snaphot.yml

```diff
@@ -0,0 +1,205 @@
+        kubectl exec $POSTGIS_POD -- apt-get update
+        kubectl exec $POSTGIS_POD -- apt-get install -y wget
+        kubectl exec $POSTGIS_POD -- mkdir -p /tmp/01
+        
+        echo "downloading snapshot"
+        
+        kubectl exec $POSTGIS_POD -- wget -O /tmp/01/pg.development.sql --no-verbose  "$SIGNED_URL"
+        
+        kubectl exec $POSTGIS_POD -- test -s /tmp/01/pg.development.sql
+
+        kubectl exec $POSTGIS_POD -- psql -U postgres -c "REVOKE CONNECT ON DATABASE postgres FROM public;"
+        echo "Terminating existing connections in postgis pod: $POSTGIS_POD"
+        
+        kubectl exec $POSTGIS_POD -- psql -U postgres -c "SELECT pid, pg_terminate_backend(pid) FROM pg_stat_activity WHERE datname = current_database() AND pid <> pg_backend_pid();"
+        echo "Dropping and creating postgres database in postgis pod: $POSTGIS_POD"
+        kubectl exec $POSTGIS_POD -- dropdb -U postgres -h localhost postgres
+
+        echo "Creating postgres database in postgis pod: $POSTGIS_POD"
+        kubectl exec $POSTGIS_POD -- createdb -U postgres -h localhost postgres
+        echo "Copying SQL files to /docker-entrypoint-initdb.d in postgis pod: $POSTGIS_POD"
+        echo "Running init-db.sh in postgis pod: $POSTGIS_POD"
+          
+        kubectl exec $POSTGIS_POD -- psql -U postgres -d hydroshare -f /init/00/init.sql
+        kubectl exec $POSTGIS_POD -- psql -U postgres -d postgres -f /tmp/01/pg.development.sql
+        #---
+        # The following commands scale the specified Kubernetes statefulsets back to their original number of replicas,
+        # effectively restarting the associated pods. This is done after completing database operations to restore
+        # normal application functionality.
+        kubectl scale $(kubectl get statefulset -o name | grep defaultworker) --replicas=$DEFAULTWORKER_REPLICAS
+        kubectl scale $(kubectl get statefulset -o name | grep hydroshare) --replicas=$HYDROSHARE_REPLICAS
+        
+        
+        # Restart the container in the postgis pod
+        echo "Restarting postgis pod: $POSTGIS_POD"
+        kubectl delete pod $POSTGIS_POD -n $NAMESPACE
+        kubectl wait --for=delete pod/$POSTGIS_POD -n $NAMESPACE --timeout=120s
+        POSTGIS_POD=$(kubectl get pods -l app=postgis -n $NAMESPACE -o jsonpath="{.items[0].metadata.name}")
+        echo "Waiting for postgis pod to restart"
+        kubectl wait --for=condition=ready pod/$POSTGIS_POD -n $NAMESPACE --timeout=180s
+        HYDROSHARE_POD=$(kubectl get pods -n $NAMESPACE -l app=hydroshare -o jsonpath="{.items[0].metadata.name}")
+        echo "Hydroshare pod: $HYDROSHARE_POD"
+        echo "wait for pod to spin up"
+        kubectl wait --for=condition=ready pod/$HYDROSHARE_POD -n $NAMESPACE --timeout=180s
+        echo "Running solr_update in Hydroshare pod: $HYDROSHARE_POD"
+        kubectl exec -n $NAMESPACE $HYDROSHARE_POD -- python manage.py solr_update
```

Comment:
consider adding a boolean workflow input "update solr?" that defaults to FALSE to so that the user can decide whether they want to update Solr.

Solr is going away soon, that's why I'm recommending default to False (also because it takes a long time and we often don't care too much if beta discover is stale)